api/structure/schema.py

```diff
@@ -0,0 +1,48 @@
+import graphene
+from graphene_sqlalchemy import SQLAlchemyObjectType
+
+from api.structure.models import Structure as StructureModel
+from helpers.auth.authentication import Auth
+from helpers.auth.error_handler import SaveContextManager
+from utilities.validations import (
+    validate_empty_fields,
+)
+
+
+class Structure(SQLAlchemyObjectType):
+    """Autogenerated Return type for OfficeStructure"""
+
+    class Meta:
+        model = StructureModel
+
+
+class CreateOfficeStructure(graphene.Mutation):
+    """ Returns payload on structure creation"""
+    class Arguments:
+        web_id = graphene.String(required=True)
+        level = graphene.Int(required=True)
+        name = graphene.String(required=True)
+        parent_id = graphene.Int(required=True)
+        tag = graphene.String(required=True)
+        location_id = graphene.Int(required=True)
+        position = graphene.String(required=True)
+
+    structure = graphene.Field(Structure)
+
+    @Auth.user_roles('Admin')
+    def mutate(self, info, **kwargs):
+        validate_empty_fields(**kwargs)
```

Comment:
We don't have to do validations for the structure fields at the moment. The data that is being saved is coming in as a list of objects, so handle this mutation in that direction.

Reply:
Also always ensure to update your branch with the base branch before raising a PR

Reply:
Okay, I will do that

Reply:
Done